Documentation/Storage-Configuration/Advanced/ceph-configuration.md

```diff
@@ -208,7 +208,7 @@ ceph osd pool set rbd pg_num 512
 
 ## Custom `ceph.conf` Settings
 
-!!! warning
+!!! info
```

Comment:
I think in keeping with pushing people to use the CephCluster cephConfig option would be better to keep it as a warning.

Reply:
To me, a warning would be for features that are being deprecated, or there is some real reason not to use it. In this case, the feature won't be deprecated since it's still needed in some scenarios (for mons, before they are in quorum). And there is no real downside for users if they are already using the feature, to keep using it if they prefer.